ethminer/MinerAux.h

```diff
@@ -314,6 +314,18 @@ class MinerCLI
 			}
 		else if (arg == "--list-devices")
 			m_shouldListDevices = true;
+		else if ((arg == "-k" || arg == "--kernel") && i + 1 < argc)
+		{
```

Comment:
since this is OpenCL Specifiy i would call it somthing like the other ocl params.